### PR TITLE
[solana] add magic eden mmm trades

### PR DESCRIPTION
**Pull request by tsekityam**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:
- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:
- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
    - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
    - **_Note_**: Please navigate to [tokens](https://github.com/duneanalytics/spellbook/tree/main/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](https://github.com/duneanalytics/spellbook/tree/main/docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:
- What does each property in the [model config block](https://github.com/duneanalytics/spellbook/blob/main/docs/models/model_config_block.md) mean?
- What is the [CI test](https://github.com/duneanalytics/spellbook/blob/main/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](https://github.com/duneanalytics/spellbook/blob/main/docs/general/best_practices.md)?

Please navigate through the [docs](https://github.com/duneanalytics/spellbook/tree/main/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

**Comment by tsekityam** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
welcome to spellbook!

i've marked this as a `wip` as it's failing the CI tests attached. if you have questions, feel free to tag @Hosuke for help

File: models/_sector/nft/trades/chains/solana/platforms/_schema.yml [hunk 1]
```diff
@@ -159,6 +159,82 @@ models:
...
+            - block_slot
+            - tx_id
+            - outer_instruction_index
+            - inner_instruction_index
+          check_columns:
+            - buyer
+            - seller
+            - account_mint
+            - amount_raw
+            - taker_fee_amount_raw
```
Comment (Hosuke) [inline models/_sector/nft/trades/chains/solana/platforms/_schema.yml:192]: The current dbt failure may be caused by the `double` typed comparison:
https://dune.com/queries/3440103

You can remove those double typed columns in `check_columns`.

Comment (tsekityam) [reply models/_sector/nft/trades/chains/solana/platforms/_schema.yml]: Thanks @Hosuke , I removed taker and maker fee from the checked column list . PTAL.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tsekityam 

[Sample magiceden_mmm_solana_trades](https://dune.com/queries/3446115)

**Review by jeff-dude — APPROVED** [review]
pushed a quick fix for prices CTE / joins, to ensure incremental reads and jinja syntax is used. otherwise, looks good and follows standards from `v2` existing spell 🤝

**Pull request merged**